js/utils.js

```diff
@@ -1132,7 +1132,40 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
 
           return bValue - aValue;
       }
-    })
+    });
+  }
+
+  function sortItems(options) {
+    var records = sortByField(options);
+
+    sortHTMLElements({
+      $layoutContainer: options.$container,
+      listContainer: options.listContainer,
+      records: records
+    });
+  }
+
+  function sortHTMLElements(options) {
+    var sortedItemsList = [];
+    var $list = options.$layoutContainer.find(options.listContainer);
```

Comment:
The way you've used `.detach()` in a loop at the moment will result in each list item being detached one after another. This is CPU heavy for the rendering engine.

Instead, detach the parent container of the list items once, update its content (like you're currently doing), and then put the `$list` element back.

```suggestion
    var $list = options.$layoutContainer.find(options.listContainer).detach();
```